tests/util/test_dataframe_ops.py

```diff
@@ -0,0 +1,102 @@
+"""
+This module contains unit tests for the `dataframe_ops.py` module.
+"""
+
+from unittest.mock import patch
+
+import pandas as pd
+import pytest
+
+from util.dataframe_ops import parse_check
+
+
+@pytest.fixture(name="setup_csv_files")
```

Comment:
move to conftest.py?
So all fixtures are at one place?

Reply:
good idea